packaging/hudi-spark-bundle/pom.xml

```diff
@@ -204,6 +204,7 @@
                     <exclude>META-INF/*.DSA</exclude>
                     <exclude>META-INF/*.RSA</exclude>
                     <exclude>META-INF/services/javax.*</exclude>
+                    <exclude>META-INF/maven/com.fasterxml.jackson.core/jackson-databind/*</exclude>
```

Comment:
This change only removes the meta info from the bundle; the classes from jackson-databind are still included.

Reply:
@yihua Yes correct, but this change is sufficient to get rid of critical CVE issue CVE-2017-17485
.